src/dispatchers/playwrightDispatcher.ts

```diff
@@ -40,12 +43,81 @@ export class PlaywrightDispatcher extends Dispatcher<Playwright, channels.Playwr
       selectors: customSelectors || new SelectorsDispatcher(scope, playwright.selectors),
       preLaunchedBrowser,
     }, false);
-    this._object.on('incomingSocksSocket', (socket: SocksInterceptedSocketHandler) => {
-      this._dispatchEvent('incomingSocksSocket', { socket: new SocksSocketDispatcher(this, socket) });
+  }
+
+  enableTethering(port: number) {
```

Comment:
I don't like the name tethering here, because we don't use it anywhere else. I'd call it `enableLocalPortForwarding`